pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/assignment/segment/OfflineSegmentAssignmentStrategy.java

```diff
@@ -68,8 +68,8 @@
  *   </li>
  * </ul>
  */
-public class OfflineSegmentAssignment implements SegmentAssignment {
-  private static final Logger LOGGER = LoggerFactory.getLogger(OfflineSegmentAssignment.class);
+public class OfflineSegmentAssignmentStrategy implements SegmentAssignmentStrategy {
```

Comment:
This does not seem correct to me

OfflineSegmentAssignment or RealtimeSegmentAssignment is not a strategy imo

Regardless of offline v/s realtime, replica group and balanced are the concrete implementations of strategies

FYI @jasperjiaguo  since you are also going to touch related code later